<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CalculatorForm</class>
 <widget class="QWidget" name="CalculatorForm" >
  <property name="objectName" >
   <string notr="true" >CalculatorForm</string>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Calculator Builder</string>
  </property>
  <layout class="QGridLayout" >
   <property name="objectName" >
    <string notr="true" />
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="objectName" >
      <string notr="true" />
     </property>
     <property name="margin" >
      <number>1</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <layout class="QVBoxLayout" >
       <property name="objectName" >
        <string notr="true" />
       </property>
       <property name="margin" >
        <number>1</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="objectName" >
          <string notr="true" >label</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>1</x>
           <y>1</y>
           <width>46</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text" >
          <string>Input 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="inputSpinBox1" >
         <property name="objectName" >
          <string notr="true" >inputSpinBox1</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>1</x>
           <y>24</y>
           <width>46</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="objectName" >
        <string notr="true" >label_3</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>1</x>
         <y>57</y>
         <width>48</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text" >
        <string>+</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <layout class="QVBoxLayout" >
       <property name="objectName" >
        <string notr="true" />
       </property>
       <property name="margin" >
        <number>1</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2_2" >
         <property name="objectName" >
          <string notr="true" >label_2_2</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>1</x>
           <y>1</y>
           <width>35</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text" >
          <string>Result</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resultLabel" >
         <property name="objectName" >
          <string notr="true" >resultLabel</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>1</x>
           <y>24</y>
           <width>35</width>
           <height>28</height>
          </rect>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignAbsolute|Qt::AlignBottom|Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignHorizontal_Mask|Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing|Qt::AlignVCenter|Qt::AlignVertical_Mask</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" >
      <layout class="QVBoxLayout" >
       <property name="objectName" >
        <string notr="true" />
       </property>
       <property name="margin" >
        <number>1</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="objectName" >
          <string notr="true" >label_2</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>1</x>
           <y>1</y>
           <width>46</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text" >
          <string>Input 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="inputSpinBox2" >
         <property name="objectName" >
          <string notr="true" >inputSpinBox2</string>
         </property>
         <property name="geometry" >
          <rect>
           <x>1</x>
           <y>27</y>
           <width>46</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1" >
      <spacer>
       <property name="objectName" >
        <string notr="true" >horizontalSpacer_2</string>
       </property>
       <property name="geometry" >
        <rect>
         <x>55</x>
         <y>84</y>
         <width>237</width>
         <height>53</height>
        </rect>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="objectName" >
      <string notr="true" >horizontalSpacer</string>
     </property>
     <property name="geometry" >
      <rect>
       <x>351</x>
       <y>9</y>
       <width>40</width>
       <height>138</height>
      </rect>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="objectName" >
      <string notr="true" >verticalSpacer</string>
     </property>
     <property name="geometry" >
      <rect>
       <x>9</x>
       <y>153</y>
       <width>336</width>
       <height>138</height>
      </rect>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
